operators/deploy/tenant-operator/values.yaml

```diff
@@ -17,6 +17,10 @@ configurations:
     url: "https://nextcloud.crownlabs.example.com/"
     user: username
     pass: password
+  userPvcSizeBytes: 1Gi
```

Comment:
```suggestion
  userPvcSize: 1Gi
```

Reply:
To uniform it to the other parameter's name is it ok to call it `mydrivePVCsSize`?

Reply:
Absolutely